src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AvoidMultipleEnumerationsTests.cs

```diff
@@ -3324,37 +3347,42 @@ End Namespace
             await VerifyVisualBasicAsync(vbCode);
         }
 
-        [Fact]
-        public async Task TestByDefaultSupposeMethodNotEnuemratesArgument()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        [InlineData(null)]
```

Comment:
Does testing `null` add any value? Otherwise, you can just use a `bool` parameter and use `CombinatorialData` attribute instead of `InlineData` attributes.

Reply:
Ah, for `null` you test no option setting. It's fine to retain the test as is.